dotnet/src/Extensions/Reliability.Polly/Config/DefaultHttpRetryHandlerFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.Config;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Reliability.Polly.Config;
```

Comment:
Do we need all three factories in a single package? I was imagining a Reliability.Default and a Reliability.Polly where Reliability.Default would be part of the Meta kernel package with the existing implementation. It's also odd that this is in the Config namespace and while this leverages HttpRetryConfig, none of the others do.

Reply:
Good point, we don't I changed all the others to be Internal and only be usable thru the Extension method.

Added a public handler that takes a Polly Policy for more customized scenarios.